### PR TITLE
Enable anonymous usage statistics tracking by default

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
At Grafana Labs we're dogfooding the anonymous usage statistics since few weeks and, as a next step, I would like to enable it by default. This will be clearly stated in the future 2.4 release notes.

#### Which issue(s) this PR fixes or relates to

Part of #1815

#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
...
 * [CHANGE] Query-frontend: CLI flag `-query-frontend.align-querier-with-step` has been deprecated. Please use `-query-frontend.align-queries-with-step` instead. #2840
 * [CHANGE] Distributor: change the default value of `-distributor.remote-timeout` to `2s` from `20s` and `-distributor.forwarding.request-timeout` to `2s` from `10s` to improve distributor resource usage when ingesters crash. #2728
-* [FEATURE] Introduced an experimental anonymous usage statistics tracking (disabled by default), to help Mimir maintainers make better decisions to support the open source community. The tracking system anonymously collects non-sensitive, non-personally identifiable information about the running Mimir cluster, and is disabled by default. #2643 #2662 #2685 #2732 #2733 #2735
+* [FEATURE] Introduced an anonymous usage statistics tracking (enabled by default), to help Mimir maintainers make better decisions to support the open source community. The tracking system anonymously collects non-sensitive, non-personally identifiable information about the running Mimir cluster. #2643 #2662 #2685 #2732 #2733 #2735 #2939
```
Comment (pstibrany) [inline CHANGELOG.md:13]: Note that original changelog entry is in 2.3.0-rc.1. When release-2.3 branch is merged to main again, this entry needs to be split.

Perhaps we should merge release-2.3 branch to main first, and then rebase this PR? WDYT?

File: docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ weight: 30
 
 # About Grafana Mimir anonymous usage statistics reporting
 
-Grafana Mimir includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Mimir cluster to a remote statistics server.
+Grafana Mimir includes a system that anonymously reports non-sensitive, non-personally identifiable information about the running Mimir cluster to a remote statistics server.
```
Comment (pstibrany) [inline docs/sources/operators-guide/configure/about-anonymous-usage-statistics-reporting.md:10]: "optionally" has been removed, but the feature is still optional.

**Review by pstibrany — APPROVED** [review]

**Comment by pstibrany** [comment]
Seems relevant: https://github.com/grafana/agent/issues/2114

File: CHANGELOG.md (hunk 1 above)
Comment (pracucci) [reply CHANGELOG.md:13]: Talked to @treid314. We can't merge `release-2.3` to `main` right now. I will help fixing the CHANGELOG once we'll do the merge next week.

**Pull request merged**